package.json

```diff
@@ -30,7 +30,7 @@
     "@dojo/has": "beta1",
     "@dojo/i18n": "beta1",
     "@dojo/shim": "beta1",
-    "maquette": "2.4.3"
+    "maquette": "2.5.0"
```

Comment:
@kitsonk Regarding resolving multiple `maquette` versions, would this scenario ever happen since `maquette` is a peer dependency of `widget-core` rather than a dependency? Either way, webpack would know that `@dojo/widget-core/node_modules/maquette/dist/maquette.js` is different from `root/node_modules/maquette/dist/maquette.js` and use different integer mids for each.